test/models/vote_test.rb

```diff
@@ -0,0 +1,69 @@
+require "test_helper"
+
+describe Vote do
+
+  describe "validations" do
+
+    it "is unique for any given work-user combination" do
```

Comment:
There is too much extra whitespace in some of your tests -- you should group by setup and execution steps but you don't need an extra line for each statement.